go.mod

```diff
@@ -15,3 +15,5 @@ require (
 	github.com/konveyor/controller v0.8.0
 	github.com/konveyor/tackle2-hub v0.0.0-20221116220406-c0054328257d
 )
+
+replace github.com/konveyor/tackle2-hub => github.com/jortel/tackle2-hub v0.0.0-20221130141916-7ae4f9a88ab9
```

Comment:
My understanding from the comment was this will be updated when
https://github.com/konveyor/tackle2-hub/pull/179 is merged.